plugin-gcal.json

```diff
@@ -0,0 +1,102 @@
+{
+    "id": "com.mattermost.gcal",
+    "name": "Google Calendar",
+    "description": "Google Calendar Integration",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-mscalendar",
+    "support_url": "https://github.com/mattermost/mattermost-plugin-mscalendar/issues",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-mscalendar/releases/tag/v1.0.0",
+    "icon_path": "assets/profile-gcal.svg",
+    "version": "1.0.0",
```

Comment:
I was thinking about versioning, and I think we should have the two plugin builds follow the same versioning. That way we can track what master changes they both have. 0/5 on this atm

Reply:
Unsure about this. My gut tells me that we should do individual versioning, and two releases when the logic is common to both, but also 0/5 on this.